flight/Modules/Airspeed/airspeed.c

```diff
@@ -104,10 +104,10 @@ static int8_t airspeedADCPin=-1;
 // Private functions
 static void airspeedTask(void *parameters);
 void baro_airspeedGet(BaroAirspeedData *baroAirspeedData, uint32_t *lastSysTime, uint8_t airspeedSensorType, int8_t airspeedADCPin);
-static void AirspeedSettingsUpdatedCb(UAVObjEvent * ev);
+static void AirspeedSettingsUpdatedCb(UAVObjEvent * ev, void *ctx, void *obj, int len);
```

Comment:
I don't think 4 parameters is a long argument list.  I am against this.

1. It is an even-greater amount of error-prone manual adjustment to existing code in the same PR to put ev into that.
2. It's really important to keep the user-context pointer as a separate argument.
3. It increases marshalling and unmarashalling for no reason.  ARM ABI dedicates 4 registers for parameter lists before chewing stack, so the microarchitecture even dedicates the most core resource (registers) assuming that this many arguments are common.